config/initializers/business_time.rb

```diff
@@ -6,3 +6,17 @@
 #  BusinessTime::Config.beginning_of_workday = "10:00 am"
 #  BusinessTime::Config.end_of_workday = "11:30 am"
 #  BusinessTime::Config.holidays << Date.parse("August 4th, 2010")
+
+module Bops
+  module TimeExtensions
+    module ClassMethods
+      def next_immediate_business_day(time)
+        Time.roll_forward(time).to_date
+      end
```

Comment:
wouldn't it be easier to use the first_business_day the gem gives us, which achieves the same? would be less code to call it:
https://github.com/bokmann/business_time/blob/17ce786e4fde7ec19b1a6f2426d69aee3acb0068/lib/business_time/time_extensions.rb#L75

Reply:
I tried using it but the `Time.first_business_day` doesn't work like I expect it to:

```ruby
irb(main):016:0> Time.first_business_day(Time.zone.now.change(hour: 03))
=> Mon, 29 Nov 2021 03:00:00.000000000 GMT +00:00
irb(main):017:0> Time.first_business_day(Time.zone.now.change(hour: 12))
=> Mon, 29 Nov 2021 12:00:00.000000000 GMT +00:00
irb(main):018:0> Time.first_business_day(Time.zone.now.change(hour: 23))
=> Mon, 29 Nov 2021 23:00:00.000000000 GMT +00:00
irb(main):019:0> 
```

It's not business-hours aware.

Reply:
ahh you're right, means the business_time config isn't loading the beginning and end of workdays properly... had a look if someone had raised a bug in the gem and its in the todos - https://github.com/bokmann/business_time#todo-
nice catch, makes me wonder though if everywhere else we are using business time we need to patch this though

Reply:
I think it is, just this method doesn't account for it. I was in the beginning of writing an issue but the gem doesn't seem that maintained, looks like https://github.com/zendesk/biz is a potential successor but something for a rainy day...